versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/extension/DatabaseAdapterExtension.java

```diff
@@ -377,7 +377,7 @@ static DatabaseAdapter createAdapterResource(
     return applyCustomConfig;
   }
 
-  @SuppressWarnings("unchecked")
+  @SuppressWarnings({"unchecked", "TypeParameterUnusedInFormals"})
```

Comment:
nit: maybe move the unchecked annotation to the cast statement, like in the other PR